README.md

```diff
@@ -308,6 +308,32 @@ Using a very high number of connections will introduce overhead that can slow th
 
 Please note that when a file is being saved, it goes through two processes. First, the downloaded file "chunks" are pieced back together to reconstruct the original file. Second, pyEGA3 calculates the checksum of the file to confirm the file downloaded successfully. Larger files will take more time to reconstruct and validate the checksum.
 
+### The --saveto argument was renamed to --output-dir
```

Comment:
```suggestion
### --saveto argument is not recognised
```

Reply:
Suggested to word the section title as the issue, instead of the explanation, to be consistent with previous section titles.